bin/lib/book.ml

```diff
@@ -6,15 +6,20 @@ let (/) = Filename.concat
 (******************************************************************************)
 (* HTML fragments                                                             *)
 (******************************************************************************)
-let head_item : Html.item =
+let head_item ?(page_title=None) () : Html.item =
+  let site_title = "Real World OCaml" in
+  let t = match page_title with
```

Comment:
"t" is a bit of a confusing name here.  How about title?

Reply:
Also, head_item is called only once, so there's no point in making the extra argument optional.  Better to have a mandatory labeled argument, e.g., "head_time ~page_title".  Then you can pass page_title in as an explicit argument.

Reply:
page_title isn't a great name for the value here, since the title you end up computing is, when you think about it, the title of the page.  I think chapter_title might be a better name.